mevislab.github.io/content/tutorials/thirdparty.md

```diff
@@ -16,42 +16,43 @@ menu:
 # MeVisLab Tutorial Chapter VIII {#TutorialChapter8}
 
 ## Using ThirdParty software integrated into MeVisLab {#TutorialThirdParty}
-MeVisLab comes with a lot of software already integrated and ready to use. Even if these tools are not available as a module, like itk and vtk for example, they can be used via Python scripting. This chapter shall give some examples of how to use a selection of integrated ThirdParty components.
+MeVisLab is equipped with a lot of useful software right out of the box, like the Insight Segmentation and Registration Toolkit (ITK) or the Visualization Toolkit (VTK). This chapter works as a guide on how to implement some third party component's functionalities to your MeVisLab projects via Python scripting. 
```

Comment:
Maybe someting like: `This chapter works as a guide on how to use some of the third party components integrated in MeVisLab for your projects via Python scripting.`